subsys/net/lib/openthread/platform/logging.c

```diff
@@ -19,13 +19,18 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #define LOG_PARSE_BUFFER_SIZE  128
 
+#ifndef OPENTHREAD_CONFIG_LOG_OUTPUT
+#define OPENTHREAD_CONFIG_LOG_OUTPUT CONFIG_LOG_OUTPUT
```

Comment:
What is it used for? Cant find any use of it in the file.

Reply:
It is used to provide (or not) implementation of otPlatLog method by Zephyr. If we want to use Spinel backend such implementation has to be turned off, because it is already provided by ncp_base.cpp file in ncp component and multiple definition error occur.